iot-hub/Quickstarts/SimulatedDevice/Parameters.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using CommandLine;
+
+namespace SimulatedDevice
+{
+    /// <summary>
+    /// Command line parameters for the SimulatedDevice sample.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+           'p',
+           "DeviceConnectionString",
+           HelpText = "The IoT hub device connection string. This is available under the \"Devices\" in the Azure portal." +
```

Comment:
Wording is awkward (`under the "Devices" in the portal`). Under the devices what?

Reply:
`This is available under \"Devices\\deviceId\" in the Azure portal."`?